app-testsuite/src/run.ts

```diff
@@ -16,6 +16,17 @@ function log(...args: any[]) {
   console.log(new Date(), ...args);
 }
 
+async function readAll(stream: NodeJS.ReadableStream) {
+  const out: string[] = [];
+  return new Promise<string>((resolve, reject) => {
+    stream.on('data', (l) => {
+      out.push(l.toString());
+    });
+    stream.on('end', () => resolve(out.join('')));
```

Comment:
Nit: This fails on a long UTF-8 output, if a point is split between two buffers.

Reply:
Good point